pom.xml

```diff
@@ -63,7 +66,7 @@
       <version>${folio-query-tool-metadata.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.folio</groupId>
+      <groupId>com.github.folio-org</groupId>
```

Comment:
Should this be changed?

Reply:
part of the above; needed to use jitpack's repository rather than the regular FOLIO one. This is just for temporary testing in CI; I'll revert it once the lib- PR is merged.

Reply:
see https://www.jitpack.io/#folio-org/lib-fqm-query-processor/modfqmmgr-87-SNAPSHOT for more details